apps/hash-api/package.json

```diff
@@ -95,7 +95,7 @@
     "ws": "8.15.1"
   },
   "devDependencies": {
-    "@blockprotocol/graph": "0.3.3",
+    "@blockprotocol/graph": "0.3.4-canary-20240313191438",
```

Comment:
This version will conflict with other PRs but I'll sort it out before merging – hopefully can get a proper `0.3.4` released in the meantime once the other pending PR in BP merges